test/parallel/test-repl-definecommand.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+require('../common');
+
+var stream = require('stream'),
+    assert = require('assert'),
+    repl = require('repl');
+
```

Comment:
@bengl You can actually check everything locally simply by running `make -j4 test`, here `4` denotes the number of tasks to be done parallelly.